testdata/.tranor/envs.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
@aflag I believe this is just a test file, not the default for installation. @fsouza: could you confirm please